tdvt/tdvt/tdvt.py

```diff
@@ -697,12 +698,17 @@ def main():
         sys.exit(0)
     elif is_test(args):
         if args.command == 'run-file':
-            output_dir = os.getcwd()
+            rfile = Path(args.run_file)
+            output_dir = Path.cwd()
             max_threads = get_level_of_parallelization(args)
-            sys.exit(run_file(args.run_file, output_dir, max_threads, args))
+            if rfile.is_file():
+                sys.exit(run_file(rfile, output_dir, max_threads, args))
+            else:
+                logging.error("Unable to open " + rfile.__str__())
+                return -1
```

Comment:
I don't know if it matters but the other branches are doing sys.exit

Reply:
good point, thanks, I'll keep it consistent.